### ISSUE TITLE
[Feature Request] Add metrics to TensorBoard callback

### ISSUE THREAD

**Issue by corrado9999** [keras-team/keras#8045]
TensorBoard is a great tool for visualizing how your network is training, but currently the related callback can only trigger graphs, histograms and embedding generation. It would be useful to (I think) anybody to be able to specify to include the metrics (all of them), or maybe a (sub?)-list of them, or even to explicitly pass a list of functions/tensors to calculate them.

Example API:
`keras.callbacks.TensorBoard(log_dir='./logs', histogram_freq=0, batch_size=32, write_graph=True, write_grads=False, write_images=False, embeddings_freq=0, embeddings_layer_names=None, embeddings_metadata=None, metrics = ['loss', 'val_loss'])`

**Comment by hermansje** [comment]
In the last version, the evolution of the model metrics is shown on the scalars page in TensorBoard.

**Comment by gabrieldemarmiesse** [comment]
This has been implemented. Closing the issue.
